### PR TITLE
Bugfix/hw 54359 transfer to paypal account is mapped to unknown

### PR DESCRIPTION
**Pull request by oleksii-lubianyi-epam**

### CODE REVIEW

File: Sources/Model/Receipt/HyperwalletReceipt.swift [hunk 1]
```diff
@@ -215,6 +216,9 @@ public struct HyperwalletReceipt: Decodable, Equatable {
         case transferToWire = "TRANSFER_TO_WIRE"
         case wireTransferFee = "WIRE_TRANSFER_FEE"
         case wireTransferReturn = "WIRE_TRANSFER_RETURN"
+        // PatPal transfer types
+        case transferToPayPalAccount = "TRANSFER_TO_PAYPAL_ACCOUNT"
```
Comment (malaw-hw) [inline Sources/Model/Receipt/HyperwalletReceipt.swift]: there is a typo ;)

File: Sources/Model/Receipt/HyperwalletReceipt.swift [hunk 2]
```diff
@@ -215,6 +216,9 @@ public struct HyperwalletReceipt: Decodable, Equatable {
         case transferToWire = "TRANSFER_TO_WIRE"
         case wireTransferFee = "WIRE_TRANSFER_FEE"
         case wireTransferReturn = "WIRE_TRANSFER_RETURN"
+        // PatPal transfer types
```
Comment (jsaini-hw) [inline Sources/Model/Receipt/HyperwalletReceipt.swift]: typo

File: Sources/Model/Receipt/HyperwalletReceipt.swift [hunk 3]
```diff
@@ -168,9 +168,10 @@ public struct HyperwalletReceipt: Decodable, Equatable {
         case prepaidCardPinChangeFee = "PREPAID_CARD_PIN_CHANGE_FEE"
         case prepaidCardRefund = "PREPAID_CARD_REFUND"
         case prepaidCardReplacementFee = "PREPAID_CARD_REPLACEMENT_FEE"
-        case pPrepaidCardSale = "PREPAID_CARD_SALE"
+        case prepaidCardSale = "PREPAID_CARD_SALE"
```
Comment (jsaini-hw) [inline Sources/Model/Receipt/HyperwalletReceipt.swift]: move this to the last before PayPal section. This section is for Prepaid Card

Comment (oleksii-lubianyi-epam) [reply Sources/Model/Receipt/HyperwalletReceipt.swift]: Moved

File: Sources/Model/Receipt/HyperwalletReceipt.swift (hunk 2 above)
Comment (oleksii-lubianyi-epam) [reply Sources/Model/Receipt/HyperwalletReceipt.swift]: Fixed

File: Sources/Model/Receipt/HyperwalletReceipt.swift (hunk 1 above)
Comment (oleksii-lubianyi-epam) [reply Sources/Model/Receipt/HyperwalletReceipt.swift]: Fixed

File: Tests/Responses/ListPrepaidCardReceiptResponse.json [hunk 4]
```diff
@@ -43,6 +43,28 @@
         "details": {
             "cardNumber": "************7917"
         }
+    }, {
+        "journalId": "CC002F14A573",
```
[the remark this answers is not in the document]
Comment (oleksii-lubianyi-epam) [reply Tests/Responses/ListPrepaidCardReceiptResponse.json]: Moved

**Pull request merged**